### ISSUE TITLE
Homebridge to failing when installed

### ISSUE THREAD

**Issue by taw-pi** [mcchots/homebridge-mqtt-temperature#9]
I installed this NPM into the latest version of Homebridge and it caused ‘Home’ to report that all my accessories were unresponsive. I uninstalled this accessory and ‘Home’ worked as expected. I have Homebridge-mqtt-relative humidity, homebridge config-u-x and TP-Link HS-100 all installed and operating as expected except when I install homebridge-mqtt-temperature. I am running homebridge on a Raspberry Pi 2 with Raspbian Stretch. Thanks Terry

**Comment by mcchots** [comment]
Can you provide your config.json settings for both temperature and humidity?
Also can you give the output of running homebridge in debug mode with `DEBUG=* homebridge -D`

**Comment by taw-pi** [comment]
The first configuration file works as expected...
The second configuration file causes the problem with 'Home' on my iPhone 6.

The log files attached are the output from the working configuration logfile-1.txt and the other is the output of the second config file that shows temperatures being logged to the log file, but 'Home' failed.

Note: I have obfuscated the passwords.

[logfile-1.txt](https://github.com/mcchots/homebridge-mqtt-temperature/files/2034691/logfile-1.txt)
[logfile-2.txt](https://github.com/mcchots/homebridge-mqtt-temperature/files/2034692/logfile-2.txt)
.

[config.json.zip](https://github.com/mcchots/homebridge-mqtt-temperature/files/2034547/config.json.zip)
[config-2.json.zip](https://github.com/mcchots/homebridge-mqtt-temperature/files/2034702/config-2.json.zip)

**Comment by taw-pi** [comment]
I ran home bridge in debug mode for the failing system as requested and the log file is attached...

[debug output for fail state.txt](https://github.com/mcchots/homebridge-mqtt-temperature/files/2034770/debug.output.for.fail.state.txt)

**Comment by mcchots** [comment]
I can see no error that could be causing the issue. If you are getting the temperature logged, then the plugin is working.

From my experience with the Home App on iOS, I frequently get a status that homekit devices are not responding only to work fine a few minutes later.

I have yet to update my machine to the latest version of Homebridge and nodejs, so I will try that as soon as I can and see if it causes any issues.

Can you watch your logs as you open the Home app on your device and see if it shows anything?

**Comment by taw-pi** [comment]
As I expected, the accessory ‘works’ but stops Home from working. I am running iOS 11.3.1 Just in case you do discover the issue.
Thanks

**Comment by mcchots** [comment]
Ok, I have found the problem. At the moment overriding the minimum and maximum temperatures is not working right. I'm not sure what has changed yet in the way it is being handled.

Can you remove the MinValue and MaxValue lines from the config like this and try again.

```
        {
            "accessory": "mqtt-temperature",
            "name": "Living Room Temperature",
            "url": "mqtt://raspi-vpn.local",
            "topic": "Sensor/temperature",
            "username": "HomeBridge",
            "password": "XXXXXXXX"
        } 

```

**Comment by taw-pi** [comment]
Looks like that was the problem,  It works OK now, many thanks..

**Comment by mcchots** [comment]
Min and Max value setting fixed in 3fdb1a162a76dac356ee84f02bb217cb558013ef
